<!-- Buttons&axes mapping --> 
<!-- Buttons -->
<launch>
<param name = "square_btn" value = "0" type = "int" />
<param name = "cross_btn" value = "1" type = "int" />
<param name = "circle_btn" value = "2" type = "int" />
<param name = "triangle_btn" value = "3" type = "int" />
<param name = "L1_btn" value = "4" type = "int" />
<param name = "R1_btn" value = "5" type = "int" />
<param name = "L2_btn" value = "6" type = "int" />
<param name = "R2_btn" value = "7" type = "int" />
<param name = "share_btn" value = "8" type = "int" />
<param name = "options_btn" value = "9" type = "int" />
<param name = "leftJoyPress_btn" value = "10" type = "int" />
<param name = "rightJoyPress_btn" value = "11" type = "int" />
<param name = "ps_btn" value = "12" type = "int" />
<param name = "touchPadPress_btn" value = "13" type = "int" />
<!-- Axes -->
<param name = "leftRigtAxis_leftStick" value = "0" type = "int" />
<param name = "upDownAxis_leftStick" value = "1" type = "int" />
<param name = "leftRigtAxis_rightStick" value = "2" type = "int" />
<param name = "L2_axs" value = "3" type = "int" />
<param name = "R2_axs" value = "4" type = "int" />
<param name = "upDownAxis_rightStick" value = "5" type = "int" />
</launch>
